### PR TITLE
fix error posted in issue #15448

### PR DESCRIPTION
**Pull request by bugface**
Signed-off-by: bugface <alexgre@ufl.edu>

# What does this PR do?

Fixes # 15448

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@LysandreJik

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
LGTM. @jdemouth could you give a quick look? Thank you!

**Comment by jdemouth** [comment]
@LysandreJik it looks good to me as well. thanks.

File: src/transformers/models/megatron_bert/convert_megatron_bert_checkpoint.py [hunk 1]
```diff
@@ -108,7 +108,8 @@ def convert_megatron_checkpoint(args, input_state_dict, config):
         config.hidden_size = ds_args.hidden_size
         config.num_hidden_layers = ds_args.num_layers
         config.num_attention_heads = ds_args.num_attention_heads
-        config.intermediate_size = ds_args.get("ffn_hidden_size", 4 * ds_args.hidden_size)
+        # config.intermediate_size = ds_args.get("ffn_hidden_size", 4 * ds_args.hidden_size)
```
[the remark this answers is not in the document]
Comment (bugface) [reply src/transformers/models/megatron_bert/convert_megatron_bert_checkpoint.py]: just removed

**Comment by LysandreJik** [comment]
Perfect, thank you!

**Pull request merged**